src/common/uniwin.h

```diff
@@ -0,0 +1,62 @@
+#if !defined(_UNISTD_H) && defined(_MSC_VER)
+#define _UNISTD_H 1
+
+/* This is intended as a drop-in replacement for unistd.h on Windows.
+ * Please add functionality as neeeded.
+ * https://stackoverflow.com/a/826027/1202830
+ */
+
+#include <stdlib.h>
+#include <io.h>
+#include "getoptwin.h"
```

Comment:
Why do we need it as a separate file, while it is just `#include <getopt.h>`?

Reply:
The distirbuted `getopt.h` file undefines `_BEGIN_EXTERN_C` and `_END_EXTERN_C` for some reason
and this may result in errors if those macros are required below the includsion point

Reply:
@ni4 I have fixed other issues

Reply:
@rrrooommmaaa Great, thanks!